server/stream.go

```diff
@@ -839,6 +839,28 @@ func (mset *stream) isLeader() bool {
 	return true
 }
 
+// Lock should be held
+func (mset *stream) stepDown() {
+	// cancel timer to create the source consumers if not fired yet
```

Comment:
Start comments with caps.

Reply:
Now removed along with all the other cleanup stuff I had added initially